images/capi/Makefile

```
@@ -226,6 +226,7 @@ RHEL_VERSIONS			:=	rhel-7
 ROCKYLINUX_VERSIONS     :=  rockylinux-8
 UBUNTU_VERSIONS			:=	ubuntu-1804 ubuntu-2004 ubuntu-2004-efi
 WINDOWS_VERSIONS		:=	windows-2019 windows-2004 windows-2022
+OPENSUSE_VERSIONS               :=      opensuse-153-cloud
```

```suggestion
OPENSUSE_VERSIONS               :=      opensuse-153
```
Consistency between this and `build_name` in the packer config

---

I can make this change . I added the cloud suffix because unlike the most os builds, the source image of this build is a cloud image with cloud init enabled and not  a bare iso as in case of ubuntu/centos etc.